lib/resources/majors.tsx

```diff
@@ -2425,144 +2479,238 @@ const bsCS_New: Major = {
+            'and create a custom course to satisfy this requirement.',
           required_credits: 6,
-          criteria:
-            'CSCI-APPL[T]^OR^CSCI-SYST[T]^OR^CSCI-SOFT[T]^OR^CSCI-RSNG[T]',
+          criteria: 'EN Computer Science[D]',
           double_count: [
-            '<b>Lower Level Undergraduate:</b><p>500.112/113/114 Gateway Computing or AP Comp Sci A or ' +
-              'equivalent<p>601.220 Intermediate Programming</p><p>601.226 Data Structures</p><p>601.229 ' +
-              'Computer System Fundamentals</p><p>601.230 Mathematical Foundations for Computer Science, or 601.231 Automata for those grandfathered into the old major</p><p>601.433 Algorithms</p>',
-            '<b>One Team(CSCI-TEAM) designated course.</b><p> This Team course may overlap other course ' +
-              'requirements, for example to count as both Team and Software.</p>',
+            '<b>Team Requirement</b> <br /> ' +
+              'Select one course with Program of Study Tag CSCI-TEAM.',
+            '<b>Upper Level CS Credits</b> <br /> ' +
+              'At least 12 more upper level CS credits are required. ',
           ],
         },
         {
           description:
-            '<b>One Team(CSCI-TEAM) designated course.</b><p> This Team course may overlap other course ' +
-            'requirements, for example to count as both Team and Software.</p>',
+            '<b>Team Requirement</b> <br /> ' +
+            'Select one course with Program of Study Tag CSCI-TEAM.',
           required_credits: 3,
           criteria: 'CSCI-TEAM[T]',
           double_count: [
-            '<b>Upper Level Undergraduate: </b><p>12 upper level CS credits in addition to the required Algorithms course</p>',
-            '<b>2 Upper Level Classifications:</b><p>At least one upper level course in two of these four different classification</p> ' +
-              'areas: Applications(CSCI-APPL), Systems(CSCI-SYST), Software(CSCI-SOFT) and Reasoning(CSCI-RSNG)',
+            '<b>Upper Level CS Credits</b> <br /> ' +
+              'At least 12 more upper level CS credits are required. ',
+            '<b>CS Electives</b> <br /> ' +
+              'Six additional credits of Computer Science are required.' +
+              'For an approved list of courses from other departments (maximum of 6 credits allowed), visit https://www.cs.jhu.edu/computer-science-other-courses-for-bs-degree/ ' +
+              'and create a custom course to satisfy this requirement.',
           ],
         },
       ],
     },
     {
-      name: 'Math',
-      required_credits: 16,
+      name: 'Computer Science Classifications',
+      required_credits: 6,
       min_credits_per_course: 3,
+      description:
+        'At least one course in two different classification areas (Applications, Reasoning, Software, Systems) must be chosen in addition to Theory (Algorithms).',
+      criteria: 'CSCI-APPL[T]^OR^CSCI-SOFT[T]^OR^CSCI-SYST[T]^OR^CSCI-RSNG[T]',
+      pathing: 2,
```

Comment:
double_count: [
        'Mathematics',
        'Humanities/Social Sciences',
        'Writing Intensive',
]

It's not a problem in frontend, but in backend there's no guarantee that distributions are processed in this order. So it is safer if all double_count fields are bidirectional. if A can double count with B, indicate that B can also double count with A. 

optional at this point but would appreciate it :)

Reply:
Added and addressed